packages/react-storage/src/components/StorageBrowser/controls/context.ts

```diff
@@ -2,7 +2,9 @@ import { createContextUtilities } from '@aws-amplify/ui-react-core';
 import { ControlsContext } from './types';
 
 const defaultValue = {
-  data: {},
+  data: {
+    title: 'StorageBrowser',
```

Comment:
Hmm I think it's debatable whether or not there should be a default title but I think we should just leave it out for now